CMakeLists.txt

```diff
@@ -48,8 +48,13 @@ endif()
 
 option(CUB_ENABLE_HEADER_TESTING "Test that all public headers compile." ON)
 option(CUB_ENABLE_TESTING "Build CUB testing suite." ON)
+set(CUB_TEST_VARIANT "ALL" CACHE STRING "Specify which CUB test variants should be built (options: ALL, THOROUGH, BENCHMARK, MINIMAL).")
 option(CUB_ENABLE_EXAMPLES "Build CUB examples." ON)
 
+if (NOT CUB_TEST_VARIANT MATCHES "^(ALL|THOROUGH|BENCHMARK|MINIMAL)$")
```

Comment:
This is moot if we switch to booleans, but FYI there's a better way to do this:

```
  set_property(CACHE CUB_TEST_VARIANT PROPERTY STRINGS
    ALL THOROUGH BENCHMARK MINIMAL
  )
```

This will populate dropdown menus, etc in CMake GUIs.

Reply:
Done.